src/main/java/edu/neu/khoury/cs5500/dijkstraspizza/config/DataLoader.java

```diff
@@ -60,23 +57,23 @@
   // === Pizzas ===
   // Lists contain regular pizzas (sm, med, lg) and gluten-free (sm, med, lg).
   private List<Pizza> cheese = new ArrayList<>(
-      Arrays.asList(new Pizza(SMALL), new Pizza(MEDIUM), new Pizza(LARGE),
-          new Pizza(SMALL), new Pizza(MEDIUM), new Pizza(LARGE)));
+      Arrays.asList(new Pizza(edu.neu.khoury.cs5500.dijkstraspizza.model.PizzaSize.small(8.0)), new Pizza(edu.neu.khoury.cs5500.dijkstraspizza.model.PizzaSize.medium(10)), new Pizza(PizzaSize.large(12)),
+          new Pizza(edu.neu.khoury.cs5500.dijkstraspizza.model.PizzaSize.small(8.0)), new Pizza(edu.neu.khoury.cs5500.dijkstraspizza.model.PizzaSize.medium(10)), new Pizza(PizzaSize.large(12))));
```

Comment:
I think the import path in these constructors should be moved up to the imports section, like the 3rd and 6th pizzas here.